### PR TITLE
feat: main component with socials

### PR DESCRIPTION
**Pull request by idzeravianka**
added main component

### CODE REVIEW

File: js_machine_front/src/components/RecentEvents.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import '../styles/recentEvents.css';
+
+interface Post {
+    id: string;
+    date: string;
+    title: string;
+    description: string;
+}
+
+const RecentEvents: React.FC<any> = (props) => {
```
Comment (vharadkou) [inline js_machine_front/src/components/RecentEvents.tsx]: don't use any

File: js_machine_front/src/routes/Main.tsx [hunk 2]
```diff
@@ -2,6 +2,9 @@ import React, { memo } from 'react';
 import Background from '../images/main.jpg';
 import '../styles/main.css';
 
+import LinkToSicials from '../components/LinkToSocials';
```
Comment (vharadkou) [inline js_machine_front/src/routes/Main.tsx]: use imports without '../' just 'components/LinksToSocials'

**Review by vharadkou — CHANGES_REQUESTED** [review]

File: js_machine_front/src/styles/linkToSocials.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.imgStyle {
```
Comment (vharadkou) [inline js_machine_front/src/styles/linkToSocials.css]: use kebab-case for naming

File: js_machine_front/src/styles/recentEvents.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
+.recentEvents__post{
```
Comment (vharadkou) [inline js_machine_front/src/styles/recentEvents.css]: kebab-case

**Review by vharadkou — CHANGES_REQUESTED** [review]

**Comment by vharadkou** [comment]
[Landing Page (Main page)](https://trello.com/c/BvPRUVzc/8-landing-page-main-page)

**Comment by idzeravianka** [comment]
[Landing Page (Main page)](https://trello.com/c/BvPRUVzc/8-landing-page-main-page)

File: js_machine_front/src/components/LinkToSocials.tsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+const sectionStyle = {
+    height: '100vh',
+    position: 'absolute' as 'absolute',
+    justifyContent: 'center',
+    display: 'flex',
+    flexFlow: 'column nowrap',
+    right: '10vh',
+};
+
+const LinkToSocials: React.FC = () => {
```
Comment (vharadkou) [inline js_machine_front/src/components/LinkToSocials.tsx]: don't use default export, always export const at the beginning

Comment (vharadkou) [inline js_machine_front/src/components/LinkToSocials.tsx]: use memo for component

File: js_machine_front/src/components/RecentEvents.tsx [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    date: string;
+    title: string;
+    description: string;
+}
+
+interface Props {
+    test: Array<Post>;
+}
+
+const RecentEvents: React.FC<Props> = (props: Props) => {
```
Comment (vharadkou) [inline js_machine_front/src/components/RecentEvents.tsx]: the same, default, memo

**Review by vharadkou — CHANGES_REQUESTED** [review]

File: js_machine_front/src/routes/Main.tsx [hunk 7]
```diff
@@ -10,8 +13,40 @@ const sectionStyle = {
...
+  },
+];
+
 export const Main: React.FC = memo(() => {
   return (
-    <div style={ sectionStyle }/>
+    <div style={sectionStyle}>
+      <LinkToSocials />
+      <div className="main__events">
+        <RecentEvents test={testPosts} />
```
Comment (vharadkou) [inline js_machine_front/src/routes/Main.tsx]: please rename test props to events

**Review by vharadkou — CHANGES_REQUESTED** [review]

File: js_machine_front/src/components/LinkToSocials.tsx [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+const sectionStyle = {
+    height: '100vh',
+    position: 'absolute' as 'absolute',
+    justifyContent: 'center',
+    display: 'flex',
+    flexFlow: 'column nowrap',
+    right: '10vh',
+};
+
+export const LinkToSocials: React.FC = memo(() => {
```
Comment (vharadkou) [inline js_machine_front/src/components/LinkToSocials.tsx]: rename to SocialLinks

Comment (vharadkou) [reply js_machine_front/src/components/LinkToSocials.tsx]: and all related styles

**Review by vharadkou — APPROVED** [review]

**Pull request merged**